release-management.md

```diff
@@ -0,0 +1,148 @@
+
+### Major Releases
+
+The go-cose project will issue major releases when substantial functionality has changed, including:
+
+- New capabilities consumers may want to opt-into, rather than automatically get as a result of getting the latest minor release or patch.
+New and unexpected capabilities _can_ be an attack vector.
+Making explicit opt-in decisions to upgrade, or possibly downgrade should be the choice of a project consuming go-cose.
+- Breaking Changes, where projects should make an implicit decision to upgrade, and adjust for breaking changes.
+
+Each major release will go through one or more `-alpha.n` and `-rc.n` pre-release phases.
+
+### Minor Releases
+
+The go-cose project will issue minor releases when incremental improvements, or bug fixes are added to existing functionality in a [Major Release](#major-releases).
+Minor releases will increment the minor field within the [semver][sem-ver] format.
+
+Each minor release will go through one or more `-alpha.n` and `-rc.n` pre-release phases.
+
+### Patch Releases
+
+Patch Releases include bug and security fixes.
+Patches will branch from the released branch being patched.
+Fixes completed in main may be ported to a patch release if the maintainers believe the effort is justified by requests from the go-cose community.
+If a bug fix requires new incremental, non-breaking change functionality, a new minor release may be issued.
+
+Principals of a patch release:
+
+- Should be a "safe bet" to upgrade to.
+- No breaking changes.
+- No feature or surface area changes.
+- A "bug fix" that may be a breaking change may require a major release.
+- Applicable fixes, including security fixes, may be cherry-picked from main into the latest supported minor release-X.Y branches.
+- Patch releases are cut from a release-X.Y.Z branch.
+
+Each patch release will go through one or more `-alpha.n` and `-rc.n` pre-release phases.
+
+### Pre-Release
+
+As builds of main become stable, and a pending release is planned, a pre-release build will be made.
+Pre-releases go through one or more `-alpha.n` releases, followed by one or more incremental `-rc.n` releases.
+
+- **alpha.n:** `X.Y.Z-alpha.n`
+  - alpha release, cut from the branch where development occurs.
+To minimize branch management, no additional branches are maintained for each incremental release.
+  - Considered an unstable release which should only be used for early development purposes.
+  - Released incrementally until no additional issues and prs are made against the release.
+  - Once no triaged issues or prs are scoped to the release, an rc is cut.
+  - To minimize confusion, and the risk of an alpha being widely deployed, alpha branches and released binaries may be removed at the discretion, and a [two-thirds supermajority][super-majority] vote, of the maintainers.
+Maintainers will create an Issue, and vote upon it for transparency to the decision to remove a release and/or branch.
+  - Not supported
```

Comment:
?

Reply:
Can I get a wee bit more about which part is the question? 
This section is focused on deleting alphas and rcs, as a forcing function (when needed) to an actual release. 
We know that orgs wind up depending on pre-release builds, and we could break someone by removing a pre-release build, so I'm just putting some process around this.

Reply:
I think, the comment is for line 93, What is the context of `-Not supported` ?

Reply:
I can add a link to the ["support"](https://github.com/veraison/go-cose/pull/188/files#diff-0f9f71fb66352a76730712a4204600abd7a6f46b400b4345886ab61cf30c45b4R105) section

Reply:
Added a link to the not-supported line. Pease unresolve if it's something else.